tasks/post-deploy/09-sitemap.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+
+if [[ -n "${APP_HOSTPATH}" ]]; then
+  echo "\"^/${APP_HOSTPATH}/(.*)sitemap.xml\",\"https://${APP_HOSTNAME}/${APP_HOSTPATH}/\$1wp-sitemap.xml\"" >sitemap.csv
+else
+  echo "\"^/(.*)sitemap.xml\",\"https://${APP_HOSTNAME}/\$1wp-sitemap.xml\"" >sitemap.csv
```

Comment:
This ends up in redirection loop locally, to `wp-wp-wp-...` because `(.*)sitemap.xml` catches `wp-sitemap.xml`. Adding a `/` to the capture `(.*/)` fixes it for me
```sh
if [[ -n "${APP_HOSTPATH}" ]]; then
  echo "\"^/${APP_HOSTPATH}/(.*/)sitemap.xml\",\"https://${APP_HOSTNAME}/${APP_HOSTPATH}/\$1wp-sitemap$
else
  echo "\"^/(.*/)sitemap.xml\",\"https://${APP_HOSTNAME}/\$1wp-sitemap.xml\"" >sitemap.csv
fi

```